### ISSUE TITLE
Keyword completion in :keys destructuring

### ISSUE THREAD

**Issue by guv** [cursive-ide/cursive#1825]
Cursive has a good completion assistance for keywords already.
Only in map destructuring the keyword completion is not available, e.g. `{:keys [my-keyw...]}`.
It would be nice if this could be added. :)

**Comment by madvas** [comment]
yeah, that'd be nice, but meanwhile you can use form like `{:keys [:my-keyw...]}` and you get autocompletion ;)

**Comment by marcus-zippy** [comment]
Adding my +1 here, this would be great.
